src/lib.rs

```diff
@@ -322,6 +323,14 @@ pub fn workspaces(module: &mut Module<i64>) -> String {
         let space_name = space.name.clone().unwrap();
         let mut space_icon = space_name.clone();
 
+        if WS_ONLY_LAST_DIGIT {
+            let n : u32 = space_icon.parse().expect(
+                "Workspace name is not a number. Please set WS_NUMBERS to\
```

Comment:
```suggestion
                "Workspace name is not a number. Please set WS_ONLY_LAST_DIGIT to\
```
Is this more accurate?